components/actionbutton/index.css

```diff
@@ -45,6 +38,13 @@
 				forced-color-adjust: none;
 			}
 		}
+
+		&:disabled,
```

Comment:
This is a great move to ensure disabled styles always take precedence over other states. I can see that this is why the disabled+selected variant is now showing disabled colors in high contrast mode. Great job here.